TACC/Core-CMS-Custom
# [APCD][React] User friendly date time display in UI

## Overview

User friendly date time display in UI

## Related
- [WP-719](https://tacc-main.atlassian.net/browse/WP-719)
## Changes
1. Use a datetime util
2. update display location for it.

## Testing

Check date listing in

1. View User Modal in : http://localhost:8000/administration/view-users/
2. Edit User Modal in : http://localhost:8000/administration/view-users/
3. Exception Listing: http://localhost:8000/administration/list-exceptions/
4. Extension Listing: http://localhost:8000/administration/list-extensions/
5. Submitter Listing for Admin: http://localhost:8000/administration/list-submissions/
6. User Submitter Listing: http://localhost:8000/submissions/list-submissions/
